### ISSUE TITLE
Foot Ulcer Segmentation Challenge

### ISSUE THREAD

**Issue by aficionadoai** [uwm-bigdata/wound-segmentation#3]
Is the validation folder the test folder? Is it setup the same as Medetec_foot_ulcer_224?  For example,  `Foot Ulcer Segmentation Challenge`  has train and validation. Is the test set being referred to as validation?

**Comment by Pele324** [comment]
No the setup is different. We have a testing dataset ready to be published later. Here is our roadmap: 
Training data: released
Validation data: released
Testing data release: July 2021
Submission deadline: 15/07/2021

**Comment by Pele324** [comment]
> Is the validation folder the test folder? Is it setup the same as Medetec_foot_ulcer_224? For example, `Foot Ulcer Segmentation Challenge` has train and validation. Is the test set being referred to as validation?

No the setup is different. We have a testing dataset ready to be published later. Here is our roadmap:
Training data: released
Validation data: released
Testing data release: July 2021
Submission deadline: 15/07/2021
